tests/sys/.cproject

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Are these supposed to have been added?
tests/sys/.cproject
tests/sys/.project
tests/sys/.settings.language.settings.xml

Reply:
No. I will remove it.